extensions/guacamole-auth-sso/modules/guacamole-auth-sso-ssl/src/main/resources/guac-manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+
+    "guacamoleVersion" : "1.5.0",
+
+    "name"      : "SSL Authentication Extension",
+    "namespace" : "ssl",
+
+    "authProviders" : [
+        "org.apache.guacamole.auth.ssl.SSLAuthenticationProvider"
+    ],
+
+    "css" : [
+        "styles/sso-providers.css"
+    ],
+
+    "js" : [ "ssl.min.js" ],
+
+    "html" : [
+        "html/sso-providers.html",
+        "html/sso-provider-ssl.html"
+    ],
+
+    "translations" : [
+        "translations/ca.json",
```

Comment:
Where are all these translations? I don't see any of these files (even `en.json`) in the `guacamole-auth-sso-ssl` package.

Reply:
These also are inherited from "guacamole-auth-sso-base": https://github.com/apache/guacamole-client/tree/master/extensions/guacamole-auth-sso/modules/guacamole-auth-sso-base/src/main/resources/translations